tests/unit_tests/forward_model_runner/test_job_dispatch.py

```diff
@@ -328,3 +331,23 @@ def test_job_dispatch_kills_itself_after_unsuccessful_job(unused_tcp_port):
             main(["script.py"])
 
         mock_killpg.assert_called_with(17, signal.SIGKILL)
+
+
+def test_killed_by_oom(tmp_path, monkeypatch):
+    """Test out-of-memory detection for pids and descendant of pids based
```

Comment:
Reformulation?
"""Test out-of-memory detection for pid and its descendants based on a mocked dmesg system utility."""

Reply:
fixed.